### PR TITLE
update show magnifier sample

### PR DESCRIPTION
**Pull request by prupani-7**
## Description

PR to update the sample to use composable mapview from the toolkit
[Versions.gradle](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/148/files#diff-d737df1dc5c860aab6f96d16a591f7d245e544d0537751fea8f139fa7abcf52c) file from [PR ](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/148) uses "arcgisToolkitVersion = '200.3.0-1'"

So, [update new module script PR](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/148) should be merged before this PR

## Links and Data

Sample Epic: `runtime/kotlin/issues/3369`

## What To Review

-  Review updated files

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

File: show-magnifier/build.gradle [hunk 1]
```diff
@@ -45,4 +45,7 @@ dependencies {
     implementation "androidx.compose.ui:ui-tooling"
     implementation "androidx.compose.ui:ui-tooling-preview"
     implementation project(path: ':samples-lib')
+    // Toolkit dependencies
+    implementation(platform("com.esri:arcgis-maps-kotlin-toolkit-bom:200.3.0-1"))
```
Comment (shubham7109) [inline show-magnifier/build.gradle:49]: Update versions.gradle file to be used here: 
```suggestion
    implementation(platform("com.esri:arcgis-maps-kotlin-toolkit-bom:$arcgisToolkitVersion"))
```

File: show-magnifier/src/main/java/com/esri/arcgismaps/sample/showmagnifier/screens/MainScreen.kt [hunk 2]
```diff
@@ -17,34 +17,37 @@
...
  * Main screen layout for the sample app
  */
 @Composable
-fun MainScreen(sampleName: String, application: Application) {
-    // create a ViewModel to handle MapView interactions
-    var mapViewModel = MapViewModel(application)
+fun MainScreen(sampleName: String) {
+    // Create an ArcGISMap and Viewpoint
+    val arcGISMap = ArcGISMap(BasemapStyle.ArcGISTopographic)
+    val viewpoint = Viewpoint(34.056295, -117.195800, 1000000.0)
```
Comment (shubham7109) [inline show-magnifier/src/main/java/com/esri/arcgismaps/sample/showmagnifier/screens/MainScreen.kt:40]: The set operation for viewpoint might look unclear with `Set(viewpoint = viewpoint)`, maybe rename the variable or  create the viewpoint object directly at the set operation. 

```suggestion
    val californiaViewpoint = Viewpoint(34.056295, -117.195800, 1000000.0)
```

**Review by shubham7109 — COMMENTED** [review]
I think it would be helpful to add some context to the readme regarding the geocompose module. Let's add a line to the `Additional Information` section:

```
This sample uses the GeoCompose Toolkit module to be able to implement a Composable MapView. 
```

And add the tags: `geocompose, tookit`

We can add this to the list of tasks for rest of the sample updates. cc @rchintapalli

**Comment by prupani-7** [comment]
Thank you, I applied the suggestions. @shubham7109

**Review by shubham7109 — COMMENTED** [review]
Might need to update the `snippets` tag in the metadata.json file.

**Comment by prupani-7** [comment]
Updated, the metadata json file. Thanks for the catch!

**Comment by prupani-7** [comment]
@shubham7109 merged the feature-branch checker changes into this branch. please re-review?

**Review by shubham7109 — APPROVED** [review]
Looks good 👍

**Pull request merged**